vulcanai2/datasets/utils.py

```diff
@@ -154,41 +154,39 @@ def stitch_datasets(df_dict, df_main=None, merge_on_columns=None,
         (default None)
     index_list: list of strings
         columns to establish as index for final stitched dataset (default None)
-
+    kwargs : keyword parameter, value is dataframe
```

Comment:
could you update this to be `dataset_dict` please?

Reply:
@rfratila Updated